src/eventTracking.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file Documents event tracking name space
+ *
+ * Event names should fallow the convention of:
+ * <project name>.<product name>.<location>.<action>
+ *
+ * @example edx.ui.admin_portal. (project) subscriptions. (product) table. (location) click (action)
+ * edx.ui.admin_portal.subscriptions.table.click
+ */
+
+/**
+ * @constant PROJECT_NAME leading project identifier for event names
+ */
+export const PROJECT_NAME = 'edx.ui.admin_portal';
+
+/**
+ * Subscription detail table events
+ */
+const SUBSCRIPTION_TABLE = `${PROJECT_NAME}.subscriptions.table`;
+export const subscriptionsTableEventNames = {
```

Comment:
nit suggestion: should there be a single default export that contains all the event names as a single object so when using it you can do something along the lines of, which might feel more like constants than mutable variables and would give a more obvious home to other events that get consolidated into this file:

```jsx
import EVENT_NAMES from './eventTracking.js';

const eventName = EVENT_NAMES.SUBSCRIPTIONS.TABLE.PAGINATION_NEXT;
```

Possible suggestion:

```jsx
export const PROJECT_NAME = 'edx.ui.enterprise.admin_portal';
export const SUBSCRIPTION_TABLE_PREFIX = `${PROJECT_NAME}.subscriptions.table`
export const SUBSCRIPTION_TABLE_EVENTS = {
  PAGINATION_NEXT: `${SUBSCRIPTION_TABLE_PREFIX}.pagination.next.clicked`,
};
export const SUBSCRIPTION_EVENTS = {
  TABLE: SUBSCRIPTION_TABLE_EVENTS,
};

const EVENT_NAMES = {
  SUBSCRIPTIONS: SUBSCRIPTION_EVENTS,
};

export default EVENT_NAMES;
```

Reply:
Good idea 👀